# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/hotfix-noopengl.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/CMakeLists.txt.vanilla	2023-03-06 19:57:51.435837509 +0100
+++ hyprland-source/CMakeLists.txt	2023-03-06 19:58:01.885837777 +0100
@@ -131,7 +131,6 @@
 target_link_libraries(Hyprland
         ${CMAKE_SOURCE_DIR}/subprojects/wlroots/build/libwlroots.so.12032 # wlroots is provided by us
         pixman-1
-        OpenGL
         GLESv2
         pthread
         ${CMAKE_THREAD_LIBS_INIT}
